room1/room1.txt

```diff
@@ -0,0 +1,10 @@
+Alex Arnaud 
+Cedric
+Peter
+Alfred
+Steve
+Abdoul
+Jen
```

Comment:
It it Jen or Jen Foko?

Reply:
It is Jen I think

Reply:
@Pierreta70  it's Jen Foko.

Reply:
Try to resolve this . I tried but I can't